src/test/java/org/jabref/logic/importer/ACMPortalParserTest.java

```diff
@@ -0,0 +1,119 @@
+
+    @BeforeEach
+    void setUp() throws URISyntaxException, MalformedURLException {
+        parser = new ACMPortalParser();
+        searchUrl = new URIBuilder("https://dl.acm.org/action/doSearch?AllField=" + searchDoi).build().toURL();
+        searchEntryList = new ArrayList<>();
+
+        BibEntry expected = new BibEntry(StandardEntryType.Conference);
+
+        expected.setField(StandardField.AUTHOR, "Tobias Olsson and Morgan Ericsson and Anna Wingkvist");
+        expected.setField(StandardField.YEAR, "2017");
+        expected.setField(StandardField.MONTH, "9");
+        expected.setField(StandardField.DAY, "11");
+        // expected.setField(StandardField.ABSTRACT, "The open source application JabRef has existed since 2003. In 2015, the developers decided to make an architectural refactoring as continued development was deemed too demanding. The developers also introduced Static Architecture Conformance Checking (SACC) to prevent violations to the intended architecture. Measurements mined from source code repositories such as code churn and code ownership has been linked to several problems, for example fault proneness, security vulnerabilities, code smells, and degraded maintainability. The root cause of such problems can be architectural. To determine the impact of the refactoring of JabRef, we measure the code churn and code ownership before and after the refactoring and find that large files with violations had a significantly higher code churn than large files without violations before the refactoring. After the refactoring, the files that had violations show a more normal code churn. We find no such effect on code ownership. We conclude that files that contain violations detectable by SACC methods are connected to higher than normal code churn.");
+        expected.setField(StandardField.SERIES, "ECSA '17");
+        expected.setField(StandardField.BOOKTITLE, "Proceedings of the 11th European Conference on Software Architecture: Companion Proceedings");
+        expected.setField(StandardField.DOI, "10.1145/3129790.3129810");
+        expected.setField(StandardField.LOCATION, "Canterbury, United Kingdom");
+        expected.setField(StandardField.ISBN, "9781450352178");
+        expected.setField(StandardField.KEYWORDS, "conformance checking, repository data mining, software architecture");
+        expected.setField(StandardField.PUBLISHER, "Association for Computing Machinery");
+        expected.setField(StandardField.ADDRESS, "New York, NY, USA");
+        expected.setField(StandardField.TITLE, "The relationship of code churn and architectural violations in the open source software JabRef");
+        expected.setField(StandardField.URL, "https://doi.org/10.1145/3129790.3129810");
+        expected.setField(StandardField.PAGETOTAL, "7");
+        expected.setField(StandardField.PAGES, "152–158");
```

Comment:
If you want, you can replace `expected.setField` by `.withField()` (which is our kind of a builder)

Reply:
fixed.